src/pages/Suppliers/index.tsx

```diff
@@ -50,7 +52,14 @@ const SuppliersPage: React.FC = () => {
   };
 
   const handleEditSupplier = (row: MRT_Row<Supplier>) => {
-    console.log("Edit supplier:", row.original);
+    setEditingSupplier(row.original);
```

Comment:
Why have you created two states for the same data ?

Reply:
Sorry for the confusion, I have removed the duplicate states for handling supplier information.